test/integration/targets/inventory/runme.sh

```diff
@@ -13,6 +13,14 @@ cleanup() {
 
 trap 'cleanup' EXIT
 
+# https://github.com/ansible/ansible/issues/61964
+ansible-playbook -i ../../inventory --limit testhost,, playbook.yml
+empty_inventory_exit_code="$?"
+if [ "$empty_inventory_exit_code" != "0" ]; then
+    echo "Inventory limit failed"
+    exit 1
+fi
+
```

Comment:
you should also check that `--limit ''` and `--limit ','` don't change the list of hosts targetted

Reply:
I added a whole slew of integration tests that also cover this scenario. Thanks for the suggestion.